tests/multithreading_tests.rs

```diff
@@ -1,64 +1,66 @@
-use redb::{Database, ReadableTable, TableDefinition};
-use std::sync::Arc;
-use std::thread;
-use tempfile::NamedTempFile;
+#[cfg(not(target_os = "wasi"))]
+mod multithreading_test {
```

Comment:
This file is the same as before. Just wrapped it in a `mod` to avoid compiling/running any of it on WASI